cmd/nginx-ingress/flags.go

```diff
@@ -198,7 +198,7 @@ var (
 	enableExternalDNS = flag.Bool("enable-external-dns", false,
 		"Enable external-dns controller for VirtualServer resources. Requires -enable-custom-resources")
 
-	includeYearInLogs = flag.Bool("include-year", false,
+	includeYearInLogs = flag.Bool("includeYear", false,
```

Comment:
This line can be removed,  as it cant be set by a user anymore. Instead, we will set it to true here
```
err = flag.Lookup("include_year").Value.Set(true)
```